aws-rds-customdbengineversion/aws-rds-customdbengineversion.json

```diff
@@ -0,0 +1,174 @@
+          "minLength": 0,
+          "maxLength": 256
+        }
+      },
+      "required": [
+        "Key",
+        "Value"
+      ],
+      "additionalProperties": false
+    }
+  },
+  "properties": {
+    "DatabaseInstallationFilesS3BucketName": {
+      "type": "string",
+      "description": "The name of an Amazon S3 bucket that contains database installation files for your CEV. For example, a valid bucket name is `my-custom-installation-files`.",
+      "minLength": 3,
+      "maxLength": 63
+    },
+    "DatabaseInstallationFilesS3Prefix": {
+      "type": "string",
+      "description": "The Amazon S3 directory that contains the database installation files for your CEV. For example, a valid bucket name is `123456789012/cev1`. If this setting isn't specified, no prefix is assumed.",
+      "minLength": 1,
+      "maxLength": 255
+    },
+    "Description": {
+      "type": "string",
+      "description": "An optional description of your CEV.",
+      "minLength": 1,
+      "maxLength": 1000
+    },
+    "Engine": {
+      "type": "string",
+      "description": "The database engine to use for your custom engine version (CEV). The only supported value is `custom-oracle-ee`.",
+      "pattern": "^[A-Za-z0-9-]{1,35}$",
+      "minLength": 1,
+      "maxLength": 35
+    },
+    "EngineVersion": {
+      "type": "string",
+      "description": "The name of your CEV. The name format is 19.customized_string . For example, a valid name is 19.my_cev1. This setting is required for RDS Custom for Oracle, but optional for Amazon RDS. The combination of Engine and EngineVersion is unique per customer per Region.",
+      "pattern": "^(11\\.\\d{1}|12\\.\\d{1}|18|19)(\\.[a-zA-Z0-9_.-]{1,50})$",
+      "minLength": 1,
+      "maxLength": 60
+    },
+    "KMSKeyId": {
+      "type": "string",
+      "description": "The AWS KMS key identifier for an encrypted CEV. A symmetric KMS key is required for RDS Custom, but optional for Amazon RDS.",
+      "pattern": "[a-zA-Z0-9_:\\-\\/]+",
+      "minLength": 1,
+      "maxLength": 2048
+    },
+    "Manifest": {
+      "type": "string",
+      "description": "The CEV manifest, which is a JSON document that describes the installation .zip files stored in Amazon S3. Specify the name/value pairs in a file or a quoted string. RDS Custom applies the patches in the order in which they are listed.",
+      "pattern": "[\\s\\S]*",
+      "minLength": 1,
+      "maxLength": 51000
+    },
+    "DBEngineVersionArn": {
+      "type": "string",
+      "description": "The ARN of the custom engine version."
+    },
+    "Status": {
+      "type": "string",
+      "description": "The availability status to be assigned to the CEV.",
+      "enum": [
+        "available",
+        "inactive",
+        "inactive-except-restore"
```

Comment:
Is this attribute case-sensitive? If not, enum will reject anything that is out of this spelling set.